fili-core/src/main/java/com/yahoo/bard/webservice/table/ConcretePhysicalTable.java

```diff
@@ -26,19 +27,21 @@
      * @param timeGrain  time grain of the table
      * @param columns  The columns for this table
      * @param logicalToPhysicalColumnNames  Mappings from logical to physical names
+     * @param metadataService datasource metadata service containing availability data for the table
      */
     public ConcretePhysicalTable(
             @NotNull TableName name,
             @NotNull ZonedTimeGrain timeGrain,
-            @NotNull Iterable<Column> columns,
-            @NotNull Map<String, String> logicalToPhysicalColumnNames
+            @NotNull Set<Column> columns,
```

Comment:
We should put this back to `Iterable`. An `Iterable` is a smaller interface than `Set` and is certainly smaller than `Collection`. The `Set` contract (ie. that there is only 1 of each column) is not needed, and the fact that a `PhysicalTable` will hold a set of columns is part of the fact that a `Table` has a `Schema`. The general property of retaining order is a good one to keep, for the same reason that unsorted arrays are harder to deal with than sorted or ordered arrays. If it's minimal extra work to keep things ordered, it's generally best to do so.